go/oasis-test-runner/scenario/e2e/basic.go

```diff
@@ -121,21 +121,51 @@ func (sc *basicImpl) Fixture() (*oasis.NetworkFixture, error) {
 			},
 		},
 		Validators: []oasis.ValidatorFixture{
-			oasis.ValidatorFixture{Entity: 1},
-			oasis.ValidatorFixture{Entity: 1},
-			oasis.ValidatorFixture{Entity: 1},
+			oasis.ValidatorFixture{
+				Entity:                     1,
+				LogWatcherHandlerFactories: DefaultBasicLogWatcherHandlerFactories,
```

Comment:
So network-wide defaults (e.g., for all nodes) are no longer supported? This seems annoying in case all the factories are the same.

Reply:
Good suggestion. I've reworked the last commit to retain network-wide default log watcher handler factories and added the ability to enable/disable them in each node fixture (with sensible defaults).